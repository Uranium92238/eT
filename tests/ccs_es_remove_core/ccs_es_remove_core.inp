system
   name: H2O He
   charge: 0
end system

do
   ground state
   excited state
end do

memory
   available: 8
end memory

solver cholesky
   threshold: 1.0d-12
end solver cholesky

solver scf
   algorithm:          scf-diis
   energy threshold:   1.0d-10
   gradient threshold: 1.0d-10
end solver scf

method
   hf
   ccs
end method

solver cc gs
   omega threshold:  1.0d-10
   energy threshold: 1.0d-10
end solver cc gs

solver cc es
   algorithm:          davidson
   singlet states:     2
   residual threshold: 1.0d-10
   energy threshold:   1.0d-10
   right eigenvectors
   remove core: {1}
end solver cc es

geometry
basis: cc-pVDZ
 H          0.86681        0.60144        5.00000
 H         -0.86681        0.60144        5.00000
 O          0.00000       -0.07579        5.00000
 He         0.10000       -0.02000        7.53000
end geometry

