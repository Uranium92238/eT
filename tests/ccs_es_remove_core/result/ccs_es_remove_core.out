


                     eT 1.6 - an electronic structure program

  ------------------------------------------------------------------------
   Author list in alphabetical order:
  ------------------------------------------------------------------------
   J. H. Andersen, A. Balbi, S. Coriani, S. D. Folkestad, T. Giovannini, 
   L. Goletto, T. S. Haugland, A. Hutcheson, I-M. Høyvik, E. F. Kjønstad, 
   H. Koch, R. Matveeva, T. Moitra, R. H. Myhre, A. C. Paul, S. Roet, 
   M. Scavino, A. K. Schnack-Petersen, A. S. Skeidsvoll, Å. H. Tveten
  ------------------------------------------------------------------------
   J. Chem. Phys. 152, 184103 (2020); https://doi.org/10.1063/5.0004713


   This is eT 1.6.0 Galaxy
  ------------------------------------------------------------
  Configuration date: 2022-02-01 15:59:21 UTC +01:00
  Git branch:         release-v1.6.0
  Git hash:           2abe95f42f8b4eabde880c038dd204ec81d587da
  Fortran compiler:   GNU 10.3.0
  C compiler:         GNU 10.3.0
  C++ compiler:       GNU 10.3.0
  LAPACK type:        SYSTEM_NATIVE
  BLAS type:          SYSTEM_NATIVE
  64-bit integers:    OFF
  OpenMP:             ON
  PCM:                OFF
  Forced batching:    OFF
  Runtime checks:     OFF
  ------------------------------------------------------------

  Calculation started: 2022-02-01 16:10:04 UTC +01:00


  :: Input file
  ================

     Note: geometry section is excluded from this print

     system
        name: H2O He
        charge: 0
     end system

     do
        ground state
        excited state
     end do

     memory
        available: 8
     end memory

     solver cholesky
        threshold: 1.0d-12
     end solver cholesky

     solver scf
        algorithm:          scf-diis
        energy threshold:   1.0d-10
        gradient threshold: 1.0d-10
     end solver scf

     method
        hf
        ccs
     end method

     solver cc gs
        omega threshold:  1.0d-10
        energy threshold: 1.0d-10
     end solver cc gs

     solver cc es
        algorithm:          davidson
        singlet states:     2
        residual threshold: 1.0d-10
        energy threshold:   1.0d-10
        right eigenvectors
        remove core: {1}
     end solver cc es


  Running on 2 OMP threads
  Memory available for calculation: 8.000000 GB


  :: RHF wavefunction
  ======================

     ==============================================================================
                                     Geometry (angstrom)
     ==============================================================================
        Atom           X                  Y                  Z         # in input
     ==============================================================================
        Basis: cc-pvdz
        1  H     0.866810000000     0.601440000000     5.000000000000        1
        2  H    -0.866810000000     0.601440000000     5.000000000000        2
        3  O     0.000000000000    -0.075790000000     5.000000000000        3
        4 He     0.100000000000    -0.020000000000     7.530000000000        4
     ==============================================================================

     ==============================================================================
                                     Geometry (a.u.)
     ==============================================================================
        Atom           X                  Y                  Z         # in input
     ==============================================================================
        Basis: cc-pvdz
        1  H     1.638033502034     1.136556880358     9.448630622825        1
        2  H    -1.638033502034     1.136556880358     9.448630622825        2
        3  O     0.000000000000    -0.143222342981     9.448630622825        3
        4 He     0.188972612457    -0.037794522491    14.229637717975        4
     ==============================================================================

  - Cholesky decomposition of AO overlap to get linearly independent AOs:

     Linear dependence threshold:             0.10E-05
     Number of atomic orbitals:               29
     Number of orthonormal atomic orbitals:   29

  - Molecular orbital details:

     Number of occupied orbitals:         6
     Number of virtual orbitals:         23
     Number of molecular orbitals:       29


  :: Hartree-Fock engine
  =========================

  Drives the calculation of the Hartree-Fock state.

  This is a RHF ground state calculation.
  The following tasks will be performed:

     1) Generate initial SAD density
     2) Calculation of reference state (SCF-DIIS algorithm)


  1) Generate initial SAD density


  2) Calculation of reference state (SCF-DIIS algorithm)

  - Setting initial AO density to sad

     Energy of initial guess:               -78.492022836361
     Number of electrons in guess:           12.000000000000

  - Screening and integral thresholds:

     Coulomb screening threshold:    0.1000E-15
     Exchange screening threshold:   0.1000E-13
     ERI cutoff:                     0.1000E-15
     One-electron integral  cutoff:  0.1000E-20
     Cumulative Fock threshold:      0.1000E+01

  - SCF solver settings:

     Maximum iterations:                   100
     Acceleration type:                   diis

  - Convergence thresholds

     Residual threshold:            0.1000E-09
     Energy threshold:              0.1000E-09

  - DIIS tool settings:

     DIIS dimension:   8

     Storage (solver scf_errors): memory
     Storage (solver scf_parameters): memory

  Iteration       Energy (a.u.)      Max(grad.)    Delta E (a.u.)
  ---------------------------------------------------------------
     1           -78.796606592568     0.9786E-01     0.7880E+02
     2           -78.828675852646     0.7077E-01     0.3207E-01
     3           -78.843487343819     0.6747E-02     0.1481E-01
     4           -78.843814479549     0.2753E-02     0.3271E-03
     5           -78.843850612079     0.3973E-03     0.3613E-04
     6           -78.843851670925     0.5220E-04     0.1059E-05
     7           -78.843851692779     0.6096E-05     0.2185E-07
     8           -78.843851693528     0.2137E-05     0.7488E-09
     9           -78.843851693630     0.3151E-06     0.1027E-09
    10           -78.843851693631     0.2637E-07     0.3979E-12
    11           -78.843851693631     0.5634E-08     0.1421E-13
    12           -78.843851693631     0.1415E-08     0.7105E-13
    13           -78.843851693631     0.6806E-09     0.4263E-13
    14           -78.843851693631     0.1491E-09     0.1421E-13
    15           -78.843851693631     0.2501E-10     0.2842E-13
  ---------------------------------------------------------------
  Convergence criterion met in 15 iterations!

  - Summary of RHF wavefunction energetics (a.u.):

     HOMO-LUMO gap:                  0.645629080219
     Nuclear repulsion energy:      12.116100574587
     Electronic energy:            -90.959952268218
     Total energy:                 -78.843851693631

  - Timings for the RHF ground state calculation

     Total wall time (sec):              0.89500
     Total cpu time (sec):               1.52047


  :: CCS wavefunction
  ======================

     Bath orbital(s):         False

   - Number of orbitals:

     Occupied orbitals:    6
     Virtual orbitals:     23
     Molecular orbitals:   29
     Atomic orbitals:      29

   - Number of ground state amplitudes:

     Single excitation amplitudes:  138


  :: Excited state coupled cluster engine
  ==========================================

  Calculates the coupled cluster excitation vectors and excitation energies

  This is a CCS excited state calculation.
  The following tasks will be performed:

     1) Cholesky decomposition of the electron repulsion integrals
     2) Preparation of MO basis and integrals
     3) Calculation of the ground state (diis algorithm)
     4) Calculation of the excited state (davidson algorithm)


  1) Cholesky decomposition of the electron repulsion integrals

   - Cholesky decomposition of electronic repulsion integrals solver
  ---------------------------------------------------------------------

  Performs a Cholesky decomposition of the two-electron electronic repulsion 
  integrals in the atomic orbital basis,

  (ab|cd) = sum_J L_ab^J L_cd^J.

  Once the Cholesky basis has been determined, the vectors L^J are constructed 
  and stored to disk. These may either be used directly, or be transformed 
  to the MO basis for use in post-HF calculations. For more information, 
  see S. D. Folkestad, E. F. Kjønstad and H. Koch, JCP, 150(19), (2019)

  - Cholesky decomposition settings:

     Target threshold is:   0.10E-11
     Span factor:           0.10E-01
     Max qual:                  1000

  - Cholesky decomposition ao details:

     Total number of AOs:                    29
     Total number of shell pairs:           120
     Total number of AO pairs:              435

     Significant shell pairs:               118
     Significant AO pairs:                  431

     Construct shell pairs:                 120
     Construct AO pairs:                    435

  Iter.  #Sign. ao pairs / shell pairs   Max diagonal    #Qualified    #Cholesky    Cholesky array size
  -------------------------------------------------------------------------------------------------------
     1               406 /     108       0.47383E+01         147             42             17052
     2               319 /      93       0.47165E-01         234            111             35409
     3               250 /      76       0.46944E-03         178            183             45750
     4               187 /      55       0.38270E-05         145            265             49555
     5                87 /      25       0.38106E-07          78            324             28188
     6                21 /       3       0.37202E-09          43            349              7329
     7                 0 /       0       0.36652E-11           7            354                 0
  -------------------------------------------------------------------------------------------------------

  - Summary of Cholesky decomposition of electronic repulsion integrals:

     Final number of Cholesky vectors: 354

 - Testing the Cholesky decomposition decomposition electronic repulsion integrals:

     Maximal difference between approximate and actual diagonal:              0.7161E-12
     Minimal element of difference between approximate and actual diagonal:  -0.1238E-14

  - Settings for integral handling:

     Cholesky vectors in memory: True
     ERI matrix in memory:       False

  - Finished decomposing the ERIs.

     Total wall time (sec):              0.41800
     Total cpu time (sec):               0.64462


  2) Preparation of MO basis and integrals


  3) Calculation of the ground state (diis algorithm)

   - DIIS coupled cluster ground state solver
  ----------------------------------------------

  A DIIS CC ground state amplitude equations solver. It uses an extrapolation 
  of previous quasi-Newton perturbation theory estimates of the next amplitudes. 
  See Helgaker et al., Molecular Electronic Structure Theory, Chapter 
  13.

  - CC ground state solver settings:

  - Convergence thresholds

     Residual threshold:            0.1000E-09
     Energy threshold:              0.1000E-09
     Max number of iterations:      100

  - DIIS tool settings:

     DIIS dimension:   8

     Storage (cc_gs_diis_errors): file
     Storage (cc_gs_diis_parameters): file

  Iteration    Energy (a.u.)        |omega|       Delta E (a.u.)
  ---------------------------------------------------------------
    1           -78.843851693631     0.3731E-10     0.7884E+02
  ---------------------------------------------------------------
  Convergence criterion met in 1 iterations!

  - Ground state summary:

     Final ground state energy (a.u.):   -78.843851693631

     Correlation energy (a.u.):            0.000000000000

     Largest single amplitudes:
     -----------------------------------
        a       i         t(a,i)
     -----------------------------------
        1      1        0.000000000000
        2      1        0.000000000000
        3      1        0.000000000000
        4      1        0.000000000000
        5      1        0.000000000000
        6      1        0.000000000000
        7      1        0.000000000000
        8      1        0.000000000000
        9      1        0.000000000000
       10      1        0.000000000000
     ------------------------------------

  - Finished solving the CCS ground state equations

     Total wall time (sec):              0.00400
     Total cpu time (sec):               0.00464


  4) Calculation of the excited state (davidson algorithm)
     Calculating right vectors

   - Davidson coupled cluster excited state solver
  ---------------------------------------------------

  A Davidson solver that calculates the lowest eigenvalues and the right 
  or left eigenvectors of the Jacobian matrix, A. The eigenvalue problem 
  is solved in a reduced space, the dimension of which is expanded until 
  the convergence criteria are met.

  A complete description of the algorithm can be found in E. R. Davidson, 
  J. Comput. Phys. 17, 87 (1975).

  - Settings for coupled cluster excited state solver (Davidson):

     Calculation type:    remove core
     Excitation vectors:  right

  - Convergence thresholds

     Residual threshold:            0.1000E-09
     Energy threshold:              0.1000E-09

     Number of singlet states:               2
     Max number of iterations:             100

   - Davidson tool settings:

     Number of parameters:                  138
     Number of requested solutions:           2
     Max reduced space dimension:           100

     Storage (cc_es_davidson_trials): file
     Storage (cc_es_davidson_transforms): file

  Iteration:                  1
  Reduced space dimension:    2

   Root  Eigenvalue (Re)   Eigenvalue (Im)    |residual|   Delta E (Re)
  ------------------------------------------------------------------------
     1   0.304660010094    0.000000000000     0.1308E+00   0.3047E+00
     2   0.412559138375    0.000000000000     0.1730E+00   0.4126E+00
  ------------------------------------------------------------------------

  Iteration:                  2
  Reduced space dimension:    4

   Root  Eigenvalue (Re)   Eigenvalue (Im)    |residual|   Delta E (Re)
  ------------------------------------------------------------------------
     1   0.283488046570    0.000000000000     0.2288E-01   0.2117E-01
     2   0.386114849763    0.000000000000     0.5873E-01   0.2644E-01
  ------------------------------------------------------------------------

  Iteration:                  3
  Reduced space dimension:    6

   Root  Eigenvalue (Re)   Eigenvalue (Im)    |residual|   Delta E (Re)
  ------------------------------------------------------------------------
     1   0.283154789468    0.000000000000     0.3316E-02   0.3333E-03
     2   0.381803631092    0.000000000000     0.1334E-01   0.4311E-02
  ------------------------------------------------------------------------

  Iteration:                  4
  Reduced space dimension:    8

   Root  Eigenvalue (Re)   Eigenvalue (Im)    |residual|   Delta E (Re)
  ------------------------------------------------------------------------
     1   0.283149206725    0.000000000000     0.2376E-03   0.5583E-05
     2   0.381537629386    0.000000000000     0.3446E-02   0.2660E-03
  ------------------------------------------------------------------------

  Iteration:                  5
  Reduced space dimension:   10

   Root  Eigenvalue (Re)   Eigenvalue (Im)    |residual|   Delta E (Re)
  ------------------------------------------------------------------------
     1   0.283149171550    0.000000000000     0.3127E-04   0.3517E-07
     2   0.381525272217    0.000000000000     0.4124E-03   0.1236E-04
  ------------------------------------------------------------------------

  Iteration:                  6
  Reduced space dimension:   12

   Root  Eigenvalue (Re)   Eigenvalue (Im)    |residual|   Delta E (Re)
  ------------------------------------------------------------------------
     1   0.283149170932    0.000000000000     0.6985E-05   0.6180E-09
     2   0.381525107728    0.000000000000     0.2165E-03   0.1645E-06
  ------------------------------------------------------------------------

  Iteration:                  7
  Reduced space dimension:   14

   Root  Eigenvalue (Re)   Eigenvalue (Im)    |residual|   Delta E (Re)
  ------------------------------------------------------------------------
     1   0.283149170727    0.000000000000     0.6118E-05   0.2052E-09
     2   0.346269701247    0.000000000000     0.1154E+00   0.3526E-01
  ------------------------------------------------------------------------

  Iteration:                  8
  Reduced space dimension:   16

   Root  Eigenvalue (Re)   Eigenvalue (Im)    |residual|   Delta E (Re)
  ------------------------------------------------------------------------
     1   0.283149170699    0.000000000000     0.9032E-06   0.2781E-10
     2   0.337304692535    0.000000000000     0.1845E-01   0.8965E-02
  ------------------------------------------------------------------------

  Iteration:                  9
  Reduced space dimension:   18

   Root  Eigenvalue (Re)   Eigenvalue (Im)    |residual|   Delta E (Re)
  ------------------------------------------------------------------------
     1   0.283149170698    0.000000000000     0.3999E-06   0.1242E-11
     2   0.336667899832    0.000000000000     0.8945E-02   0.6368E-03
  ------------------------------------------------------------------------

  Iteration:                 10
  Reduced space dimension:   20

   Root  Eigenvalue (Re)   Eigenvalue (Im)    |residual|   Delta E (Re)
  ------------------------------------------------------------------------
     1   0.283149170697    0.000000000000     0.8856E-07   0.1611E-12
     2   0.336581894053    0.000000000000     0.2098E-02   0.8601E-04
  ------------------------------------------------------------------------

  Iteration:                 11
  Reduced space dimension:   22

   Root  Eigenvalue (Re)   Eigenvalue (Im)    |residual|   Delta E (Re)
  ------------------------------------------------------------------------
     1   0.283149170697    0.000000000000     0.1182E-07   0.6828E-14
     2   0.336578991750    0.000000000000     0.3017E-03   0.2902E-05
  ------------------------------------------------------------------------

  Iteration:                 12
  Reduced space dimension:   24

   Root  Eigenvalue (Re)   Eigenvalue (Im)    |residual|   Delta E (Re)
  ------------------------------------------------------------------------
     1   0.283149170697    0.000000000000     0.1163E-08   0.2220E-15
     2   0.336578932287    0.000000000000     0.3538E-04   0.5946E-07
  ------------------------------------------------------------------------

  Iteration:                 13
  Reduced space dimension:   26

   Root  Eigenvalue (Re)   Eigenvalue (Im)    |residual|   Delta E (Re)
  ------------------------------------------------------------------------
     1   0.283149170697    0.000000000000     0.8083E-10   0.2942E-14
     2   0.336578931682    0.000000000000     0.2423E-05   0.6056E-09
  ------------------------------------------------------------------------

  Iteration:                 14
  Reduced space dimension:   27

   Root  Eigenvalue (Re)   Eigenvalue (Im)    |residual|   Delta E (Re)
  ------------------------------------------------------------------------
     1   0.283149170697    0.000000000000     0.3709E-10   0.3997E-14
     2   0.336578931679    0.000000000000     0.2559E-06   0.3112E-11
  ------------------------------------------------------------------------

  Iteration:                 15
  Reduced space dimension:   28

   Root  Eigenvalue (Re)   Eigenvalue (Im)    |residual|   Delta E (Re)
  ------------------------------------------------------------------------
     1   0.283149170697    0.000000000000     0.3712E-10   0.3886E-14
     2   0.336578931679    0.000000000000     0.3353E-07   0.4102E-13
  ------------------------------------------------------------------------

  Iteration:                 16
  Reduced space dimension:   29

   Root  Eigenvalue (Re)   Eigenvalue (Im)    |residual|   Delta E (Re)
  ------------------------------------------------------------------------
     1   0.283149170697    0.000000000000     0.3773E-10   0.2665E-14
     2   0.336578931679    0.000000000000     0.3922E-08   0.3941E-14
  ------------------------------------------------------------------------

  Iteration:                 17
  Reduced space dimension:   30

   Root  Eigenvalue (Re)   Eigenvalue (Im)    |residual|   Delta E (Re)
  ------------------------------------------------------------------------
     1   0.283149170697    0.000000000000     0.3272E-10   0.7772E-15
     2   0.336578931679    0.000000000000     0.3511E-09   0.1998E-14
  ------------------------------------------------------------------------

  Iteration:                 18
  Reduced space dimension:   31

   Root  Eigenvalue (Re)   Eigenvalue (Im)    |residual|   Delta E (Re)
  ------------------------------------------------------------------------
     1   0.283149170697    0.000000000000     0.3253E-10   0.2054E-14
     2   0.336578931679    0.000000000000     0.3887E-10   0.1665E-15
  ------------------------------------------------------------------------
  Convergence criterion met in 18 iterations!

  - Excitation vector amplitudes:

     Electronic state nr. 1

     Energy (Hartree):                  0.283149170697
     Fraction singles (|R1|/|R|):       1.000000000000

     Largest single amplitudes:
     -----------------------------------
        a       i         R(a,i)
     -----------------------------------
        1      6       -0.983646028265
        4      6       -0.157027524555
        6      6        0.065055945029
       13      6        0.047162058245
       15      6       -0.020455814185
        1      3        0.011446284420
        9      6        0.011161183869
       10      6       -0.009799973370
       22      6       -0.009444714068
       19      5        0.009159116931
     ------------------------------------

     Electronic state nr. 2

     Energy (Hartree):                  0.336578931679
     Fraction singles (|R1|/|R|):       1.000000000000

     Largest single amplitudes:
     -----------------------------------
        a       i         R(a,i)
     -----------------------------------
        2      6        0.966997222913
        3      6        0.225319759189
        7      6       -0.108836919079
       14      6        0.044308439492
        2      3       -0.011730892569
       12      6       -0.007347145592
        8      2       -0.005838857329
        3      3       -0.004760670113
        2      5        0.004643584906
        5      4       -0.003941625321
     ------------------------------------

     - Electronic excitation energies:

                                      Excitation energy
                          ------------------------------------------
      State                (Hartree)             (eV)
     ---------------------------------------------------------------
        1                  0.283149170697        7.704881385090
        2                  0.336578931679        9.158779236105
     ---------------------------------------------------------------
     eV/Hartree (CODATA 2014): 27.21138602

  - Finished solving the CCS excited state equations (right)

     Total wall time (sec):              0.08300
     Total cpu time (sec):               0.09525

  - Timings for the CCS excited state calculation

     Total wall time (sec):              0.50800
     Total cpu time (sec):               0.74713

  ------------------------------------------------------------

  Peak memory usage during the execution of eT: 10.665728 MB

  Total wall time in eT (sec):              1.47200
  Total cpu time in eT (sec):               2.33108

  Calculation ended: 2022-02-01 16:10:06 UTC +01:00

  - Implementation references:

     eT: https://doi.org/10.1063/5.0004713
     Cholesky decomposition of ERIs: https://doi.org/10.1063/1.5083802

  eT terminated successfully!
